server/pkg/pipeline/queue.go

```diff
@@ -67,6 +85,9 @@ func (q Queue[T]) Enqueue(ctx context.Context, item T) {
 			item = q.EnqueuePreprocessorFn(ctx, item)
 		}
 
+		q.enqueueCounter.Add(ctx, 1, metric.WithAttributes(
+			attribute.String("queue.name", q.name),
+		))
```

Comment:
Is there any other information available at this moment that we should include in the metric attributes?